test/components/select-test.jsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+var React = require('react');
+var TestUtils = require('react-addons-test-utils');
+var assert = require('chai').assert;
+var sinon = require('sinon');
+var SelectView = require('../../src/components/select/select.jsx');
+
+describe('SelectComponent', function() {
+  it('should render select with label', function() {
+    var select = TestUtils.renderIntoDocument(
+      <SelectView label="test" />
+    );
+
+    var renderedSelect = TestUtils.findRenderedDOMComponentWithClass(select, 'component-select-label');
+    assert.isDefined(renderedSelect);
+
+  });
+
+  it('should not have default value', function() {
+    var select = TestUtils.renderIntoDocument(
```

Comment:
So you're not able to have e.g.

describe('when SelectView has no attributes', function() {
beforeEach(function() {
 var select = TestUtils.renderIntoDocument(
   <SelectView />
  );
 var renderedSelect = TestUtils.findRenderedDOMComponentWithClass(select, 'component-select-field');
});
 it('should not have default value', function() {
  assert.equal(renderedSelect.value, '');
 });
 it('should have default ID', function() {
 assert.equal(renderedSelect.id, 'component-select');
});
it('should have default name', function() {
assert.equal(renderedSelect.name, 'component-select');
 });
});